unic/ucd/normal/src/lib.rs

```diff
@@ -54,6 +54,7 @@ mod decomposition_type;
 pub use decomposition_type::DecompositionType;
 
 mod hangul;
+pub use hangul::decompose as decompose_hangul_syllable;
```

Comment:
Would you please add a `TODO(GH-209): Drop after unic-ucd-hangul is available` here?

And please feel free to grab it yourself, or post it to This Week in Rust for new contributors.